packages/v8env/src/headers.ts

```diff
@@ -10,175 +10,116 @@ import { logger } from "./logger"
  * @class
  * @param {Object} [init]
  */
-export default function Headers(init) {
-  Object.defineProperty(this, "_headerList", {
-    enumerable: false,
-    value: []
-  })
-  if (init) {
-    fill(this, init)
+export default class Headers {
+  private headerMap: { [key: string]: string[] } = {};
```

Comment:
We can use `Map<string, string[]>` which gives native `has`, `forEach`, and iterator.